erpnext/crm/doctype/opportunity/opportunity.js

```diff
@@ -126,6 +131,13 @@ frappe.ui.form.on("Opportunity", {
 		})
 	},
 
+	make_request_for_quotation: function(frm) {
+		frappe.model.open_mapped_doc({
+			method: "erpnext.crm.doctype.opportunity.opportunity.make_request_for_quotation",
+			frm: cur_frm
```

Comment:
No need of `cur_frm`. You can use `frm` since you have it available. `cur_frm` was old pattern. Change above as well, wherever applicable

Reply:
@deepeshgarg007 Done!